pkg/cmd/admin.go

```diff
@@ -55,11 +55,19 @@ func newAdminDataCommand() *cobra.Command {
 			return nil
 		},
 	}
-	v := version.Get()
+
+	versionString := prerun.GetVersionStringOfData()
```

Comment:
If `GetVersionStringOfData` returns the error instead of treating an unknown error as version `""`, then this call could _also_ return the error and not take any action. If the version file exists but is empty or we can't read it, that data is broken and we don't want to mess with it.